src/components/NavBar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../style/navbar.css';
+
+const NavBar = () => (
+  <div className="nav">
+    <h1>Book Store</h1>
+    <ul>
+      <li>
+        <Link to="/">BOOKS</Link>
+      </li>
+      <li>
+        <Link to="/categories">CATEGORIES</Link>
+      </li>
```

Comment:
✅ Has 2 <Link>s in the navbar which leads to the booklist and categories locations 👍🏻